rasa/core/test.py

```diff
@@ -144,24 +141,23 @@ class WronglyClassifiedUserUtterance(UserUttered):
 
     type_name = "wrong_utterance"
 
-    def __init__(
-        self,
-        text,
-        correct_intent,
-        correct_entities,
-        parse_data=None,
-        timestamp=None,
-        input_channel=None,
-        predicted_intent=None,
-        predicted_entities=None,
-    ):
-        self.predicted_intent = predicted_intent
-        self.predicted_entities = predicted_entities
+    def __init__(self, event: UserUttered, eval_store: EvaluationStore):
 
-        intent = {"name": correct_intent}
+        if not eval_store.intent_predictions:
+            self.predicted_intent = None
+        else:
+            self.predicted_intent = eval_store.intent_predictions[0]
+        self.predicted_entities = eval_store.entity_predictions
+
+        intent = {"name": eval_store.intent_targets[0]}
```

Comment:
Please use a constant for `"name"`

Reply:
What do you mean?

Reply:
Create a constant in https://github.com/RasaHQ/rasa/blob/master/rasa/core/constants.py or in `rasa/core/test.py`, such as `INTENT_NAME =  "name"`. You can then use this variable everywhere, also in tests. Advantage: If you want to rename the variable, you just have to change one line.
Optional: You could do the same for all the other keyword strings in the file.

Reply:
I see why, but I think this is controversial.  There is benefit in keeping our variable names constant but this seems like it would be a confusing change for someone developing custom code with Rasa

Reply:
not sure I can follow, but we can also leave it out and tackle it / discuss it later

Reply:
SGTM 😄